src/components/SearchInput/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import ClassNames from 'classnames';
+
+import styles from '@/components/SearchInput/SearchInput.css';
+
+import IconSearch from '@/assets/images/icons/web/search-simple.svg';
+import IconClose from '@/assets/images/icons/web/close.svg';
```

Comment:
please rename it to iconClose. The capital letter should be used for components only